src/components/synopsis-report-form/synopsis-report-form.js

```diff
@@ -200,8 +200,8 @@ class SynopsisReportForm extends React.Component {
 
     const sportsUpdateOK = this.notEmpty('Weekly_Sports_and_Activities_Update__c');
 
-    const ijImageOK = sr.Identity_Journal_Status__c === 'Not turned in'
-      || (sr.Identity_Journal_Status__c === 'Turned in' && !!(this.props.imagePreviews && this.props.imagePreviews.length));
+    // const ijImageOK = sr.Identity_Journal_Status__c === 'Not turned in'
+    //   || (sr.Identity_Journal_Status__c === 'Turned in' && !!(this.props.imagePreviews && this.props.imagePreviews.length));
```

Comment:
Rather than leave zombie code around lets just delete these lines.  There are other instances of ijImageOK in the file. Those should be removed as well. (I find 5 in the file, so there are 3 left to be removed.)